src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/Editor_OutOfProc.cs

```diff
@@ -127,5 +129,22 @@ public void WaitForActiveView(string viewName)
 
         public string[] GetErrorTags()
             => _editorInProc.GetErrorTags();
+
+        public void EncapsulateField()
+        {
+            SendKeys(new KeyPress(VirtualKey.R, ShiftState.Ctrl), new KeyPress(VirtualKey.E, ShiftState.Ctrl));
+        }
+
+        public void WaitForPreviewChanges()
+            => _editorInProc.WaitForPreviewChanges();
+
+        public void ClickPreviewChangesOK()
```

Comment:
Isn't this what `EncapsulateField_OutOfProc` (/ `PreviewChanges_OutOfProc`)is for?